docs/tools/erc725js/providers.md

```diff
@@ -27,3 +23,19 @@ The following code snippet will use the web3 provider available at web3.provider
 ```javascript
 const ethereumProvider = window.ethereum;
 ```
+
+## Web3
+
+:::info Warning
```

Comment:
If warning, let's use: `caution` instead of info.

List avail. here FYI:

https://docusaurus.io/docs/markdown-features/admonitions
```suggestion
:::caution Warning
```